### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MakaronX**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,77 @@
...
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  max-width: 100%;
```
Comment (YegorKorenkov) [inline src/style.css:21]: redundant

File: src/index.html [hunk 2]
```diff
@@ -3,12 +3,57 @@
...
+        <img
+          src="./images/logo.png"
+          alt="logo_MOYO"
+          class="header__images"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
```
Comment (YegorKorenkov) [inline src/index.html:26]: image

File: src/style.css [hunk 3]
```diff
@@ -1 +1,77 @@
...
+  align-items: center;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  margin: 0;
```
Comment (YegorKorenkov) [inline src/style.css:64]: Please remove margin-left for first nav item, but not for is-active. If, for example, the third element becomes active, then its margin-right will also be deleted

File: src/style.css [hunk 4]
```diff
@@ -1 +1,77 @@
...
+  position: relative;
+}
+
+.is-active:after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 37px;
```
Comment (YegorKorenkov) [inline src/style.css:73]: make this line on fullwidth of parent

File: src/style.css [hunk 5]
```diff
@@ -1 +1,77 @@
...
+  text-align: center;
+  align-items: center;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
```
Comment (YegorKorenkov) [inline src/style.css:63]: you can also move this blue color to variable

**Review by YegorKorenkov — CHANGES_REQUESTED** [review]

**Comment by MakaronX** [comment]
Is everything okay now?

File: src/style.css (hunk 3 above)
Comment (MakaronX) [reply src/style.css:67]: I don't know how to do it

**Pull request closed without merging**